x-pack/plugins/data_usage/common/rest_types/usage_metrics.ts

```diff
@@ -123,6 +123,13 @@ export const UsageMetricsAutoOpsResponseSchema = {
       ),
     }),
 };
-export type UsageMetricsAutoOpsResponseSchemaBody = TypeOf<
+export type UsageMetricsAutoOpsResponseMetricSeries = TypeOf<
   typeof UsageMetricsAutoOpsResponseSchema.body
->;
+>['metrics'][MetricTypes][number];
+
+export type UsageMetricsAutoOpsResponseSchemaBody = Omit<
```

Comment:
I changed this type because I was getting a type error when only sending a subset of metrics, so this allows the subset instead of all.  @ashokaditya Hope thats ok